### ISSUE TITLE
Missing libraries in release

### ISSUE THREAD

**Issue by Ivorforce** [MinecraftForge/FML#651]
Port of the comment chain from d14d1a8fea4c9242c944079ab8e4cdd516dfce4c.

Since d14d1a8fea4c9242c944079ab8e4cdd516dfce4c, people have been getting crashes with my mod due to missing libraries. The active issue on my repo is https://github.com/Ivorforce/RecurrentComplex/issues/16. To my knowledge, the issue is 100% reproducible with Forge 10.13.3.1388, Recurrent Complex 0.9.6.1, and IvToolkit 1.1.1.

In short, the library containing Apache Common's Triple is missing:

```
java.lang.NoClassDefFoundError: org/apache/commons/lang3/tuple/Triple
```

Updating the launcher, as previously suggested, does not fix the issue. The crash occurs in 10.13.3.1388 and later, but not in 10.13.3.1385-1.7.10 and earlier.

**Comment by cpw** [comment]
Capture the output in the launcher please.

**Comment by Ivorforce** [comment]
Launcher 1.6.11, Forge 10.13.3.1395-1710ls, Recurrent Complex 0.9.6.1, IvToolkit 1.1.1:

https://gist.github.com/Ivorforce/d7c416ec5d8b772ebcde

Crash occurs when starting a world, which is where I reference Triple for the first time.

**Comment by cpw** [comment]
I need what is put in the launcher screen, not the log file- that shows how the launcher constructs the class path..

**Comment by Ivorforce** [comment]
That was not the log file, that was the Game Output tab from the launcher. Do you mean the 'Launcher Log' tab? In case you do, I added that to the gist (https://gist.github.com/Ivorforce/d7c416ec5d8b772ebcde#file-launcher-log-L1).

**Comment by cpw** [comment]
Yes - it contains the "half-command" as they call it, and it's not a bug. Problem has already been identified - the dev env offers apache commons lang 3.2.1, whereas Mojang seem to be using commons lang 3.1. I'm guessing Triple and friends were added to 3.2. I'll be fixing shortly, but I need to verify infrastructure first.

**Comment by Bogdan-G** [comment]
Forge normal work with libraries: 
commons-compress-1.9
commons-codec-1.10
guava-18.0
gson-2.3.1
vecmath-1.5.2
scala-library-2.11.5
scala-parser-combinators_2.11-1.0.3
lwjgl 2.9.4
akka-actor_2.11-2.3.9
P.s. Test local modpack ~100 mods.
P.p.s. But lib NettyIO(io.netty:netty-all:4.0.25.Final) and newer, crash SSP-server of error packet-handler

**Comment by LexManos** [comment]
Boddan your post makes no sense.
Logs, and a better explanation.
We do not, and will not support 3rd party modifications such as Cauldron. {They have their own support for that}
If you say there are errors then post logs. And yes Triple was added in 3.2 see: http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/Triple.html
